content/en/docs/best-practices/channel-naming.md

```diff
@@ -96,25 +96,35 @@ spec:
+for cluster admins and Operator consumers in general when each solution distribute in the OLM catalog adopts different names
+or different meanings for the same terminologies.
 
 The names you choose are notional and up to you to decide, however, picking good channel names requires some basic guidance.  What is described below are different channel naming conventions that are commonly used by the operator community to denote different operator upgrade use cases.
 
 ### Naming Conventions
 
 * Channel names are chosen by operator authors as they see fit to meet their upgrade strategies.
+* Channel names are related to the release strategy and not to specific features. ( e.g. we can have an Operator bundle release which is supported and ought to be under the stable channel but with a "tech-preview" and not supported feature. ).
+
+> Note: If your new operator bundle release contains an API(CRD), which represents new experimental features and not supported (like "tech-preview"), then
+the terminology recommended to being adopted would be in the API version (e.i. group: my.example.com  kind: Backup version: v1alpha1) and **not** in the channel naming. The recommendation for this scenario is to follow up the [Kubernetes API versioning](https://kubernetes.io/docs/reference/using-api/#api-versioning).
+
 * Channel names are unique to your operator and do not collide with channel names used by other operator providers.
 * Seldom is there a situation where your channel names need to contain information about the Kubernetes or Openshift cluster version they run on.  Only in the case where your operator versions have a dependency on the Kubernetes/Openshift version would you include the cluster version in your channel name.
-* You typically would not include product names in your channels since the channels are unique to your product and will not collide with other channel names used by other operators.
+* You typically would **not** include product names in your channels since the channels are unique to your product and will not collide with other channel names used by other operators.
 * You could include or have an operand version in your channel name to advertise to consumers the version of operand they can subscribe to.
 * If you do choose to include some version in your channel name, it is important to include an additional identifier, to clarify what that the version number is referring to. A version number could equally be referring to product version (operand version), or operator version - these two don't always match: the operator itself can have different versioning than the product it is managing. 
-
+* It is recommended to use the major versions of your Operator releases in the channel names at least. So that cluster admins can better plan when they can begin to consume newer versions of your Operators, which contains breaking changes and avoid workflow issues. (e.g. `stable-2.x`)
```

Comment:
```suggestion
* It is recommended to use the major versions of your Operator releases in the channel names at least. So that cluster admins can better plan when they can begin to consume newer versions of your Operators, which contains breaking changes and avoid workflow issues. (e.g. `stable-v2`)
```